config/arch.aarch64

```diff
@@ -11,26 +11,31 @@
 # determine architecture's family
   case $TARGET_CPU in
     generic|cortex-a35|cortex-a53|cortex-a57|cortex-a72|exynos-m1|qdf24xx|thunderx|xgene1|cortex-a57.cortex-a53|cortex-a72.cortex-a53|cortex-a73.cortex-a53)
-      TARGET_SUBARCH=aarch64
-      TARGET_VARIANT=armv8-a
+      TARGET_SUBARCH=armv8-a
       TARGET_ABI=eabi
       TARGET_FEATURES+=" neon"
       # This only makes sense for 8.0-a, later revisions have LSE
       TARGET_CFLAGS=" -mno-outline-atomics"
       ;;
-      cortex-a55|cortex-a76.cortex-a55)
-      TARGET_SUBARCH=aarch64
-      TARGET_VARIANT="${TARGET_VARIANT:-armv8.2-a}"
+    cortex-a55|cortex-a76.cortex-a55|cortex-a77)
+      TARGET_SUBARCH=armv8.2-a
+      TARGET_ABI=eabi
+      TARGET_FEATURES+=" neon"
+      ;;
+    cortex-x3)
+      TARGET_SUBARCH=armv9-a
```

Comment:
nope that doesn't works

Reply:
I'm curious what issue you encounter. I didn't have an issue when I tried on my own Odin 2.

Reply:
emulation station crashing with an invalid operator, I'll give a run with your branch

Reply:
I wouldn't be surprised if armv9-a didn't work with the odin 2 before this branch. The aarch64 compiler would have been built to use the armv9-a arch (with no flags) by default, which enables sve+sve2, neither of which are supported by SM8550.